pg_activity

```diff
@@ -275,6 +275,8 @@ server activity monitoring.")
         delta_disk_io = None
         # get DB informations
         db_info = PGAUI.data.pg_get_db_info(None, using_rds = options.rds)
+        # get active connections
+        active_connections = PGAUI.data.pg_get_active_connections()    
```

Comment:
At this point, we don't need to get active connections count, you can remove those 2 lines. Thanks.